src/eos_gates.js

```diff
@@ -178,13 +184,27 @@ function getAppStoreAppId(remote, appId) {
     return appId;
 }
 
-function installAppFromStore(remote, appId) {
+// Happens when a .exe or .msi file is opened. Contains the
+// argv that was passed to the application.
+const EVENT_LAUNCHED_INSTALLER_FOR_FLATPAK = 'e98bf6d9-8511-44f9-a1bd-a1d0518934b9';
+
+const EVENT_LAUNCHED_EXISTING_FLATPAK = '192f39dd-79b3-4497-99fa-9d8aea28760c';
+
+const EVENT_LAUNCHED_EQUIVALENT_INSTALLER_FOR_FLATPAK = '7de69d43-5f6b-4bef-b5f3-a21295b79185';
+
+const EVENT_LAUNCHED_EQUIVALENT_EXISTING_FLATPAK = '7de69d43-5f6b-4bef-b5f3-a21295b79185';
```

Comment:
Yikes, yep, must have forgotten to paste in the other generated one properly....